packages/cli/src/workflows/workflow.service.ts

```diff
@@ -456,7 +456,13 @@ export class WorkflowService {
 				select: ['id'],
 				where: { workflowId },
 			})
-			.then((rows) => rows.map(({ id: executionId }) => ({ workflowId, executionId })));
+			.then((rows) =>
+				rows.map<BinaryData.FileLocation>(({ id: executionId }) => ({
+					type: 'execution',
+					workflowId,
+					executionId,
+				})),
```

Comment:
Could we encapsulate the creation of these objects into a factory function somewhere?